terraform/install.sh

```diff
@@ -112,7 +112,14 @@ promptForProject() {
           IFS=$' |'
           opt=($opt)
           project_id=${opt[0]}
-          break
+          # check if this project has a bucket
+          bucket_name="$project_id-bucket"
```

Comment:
we shouldn't be setting this in multiple places in the code, in case we want to change it later.

Reply:
So create a temporary var?

Reply:
somehow refactor so that the variable is only set once in the script. Maybe even just declare it at the top?

Reply:
But I can only check the bucket when a project is selected, so I can't know it beforehand. Might as well create a temp var.

Reply:
Hmm good point. What do you mean by a temp var?

Reply:
Why not creating a function that gets projectId as input and returns a bucker name back and then use it everywhere where it's needed?